gdscript-godot.el

```diff
@@ -102,20 +102,26 @@ When run with prefix argument, it offers extra debug options to choose from."
 (defun gdscript-godot-run-current-scene ()
   "Run the current script file in Godot Engine. Use the universal prefix (C-u) to force a scene select."
   (interactive)
-  (gdscript-godot--run-command (gdscript-godot--select-scene current-prefix-arg)))
+  (let ((scene (gdscript-godot--select-scene current-prefix-arg)))
+    (when scene
+      (gdscript-godot--run-command scene))))
 
 (defun gdscript-godot-run-current-scene-debug ()
   "Run the current script file in Godot Engine.
 
 When run with prefix argument, it offers extra debug options to choose from."
   (interactive)
-  (gdscript-godot--debug-options-handler debug-options
-    (gdscript-godot--run-command "-d" debug-options (gdscript-godot--select-scene))))
+  (let ((scene (gdscript-godot--select-scene current-prefix-arg))) ;; TODO - debug and force select using prefix. need to tweak this sometime.
```

Comment:
Please don't leave comments and, in particular, TODOs in the code. If a feature is unfinished or has a known bug, it should be addressed before merging. If a feature can be improve, then please open a new issue.

Reply:
Prior to this fix if you hit C-g (I use projectile) it still ran the default scene whereas it should abort.   

Re todo, yes I'll address that tomorrow. End of day here.

Reply:
Gotcha, thanks much!